docker/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM phusion/passenger-full:3.0.7
+
+RUN bash -lc "rvm --default use ruby-3.3.4"
+
+RUN apt-get update && \
```

Comment:
I was thinking about writing in some short comments about the dependencies for folks skimming this. Is there any reason (convention or otherwise) to favor `&&` vs separate `RUN` statements?  (I was thinking for comments separate statements might be cleaner...